solr_init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
```

Comment:
A quick explanation here would be useful as well, similar to the above bash script

Reply:
added a short explanation